docs/src/info/audio_object.mdx

```diff
@@ -0,0 +1,22 @@
+---
+name: Audio Object
+menu: "General info"
+route: info/audio_object
+---
+
+# Audio Object
+
+API version 5.0 and higher
```

Comment:
This documentation is a bit dated. Would you mind adding more fields, at least which are more certainly known? i.e. `main_artists`, `access_key`, `is_explicit`, `no_search`... This is not crucial, of course.

Reply:
Oh, I thought these parameters were optional, since they can't be operated on, so I analyzed the data provided by the VK API and I think it's worth putting it in a separate branch, since most of the parameters are not described.